gulpfile.js

```diff
@@ -33,56 +34,68 @@ gulp.task('compile:pl', plCompile);
  */
 const twigNamespaces = require('./tools/tasks/gulp-twig-namespaces');
 
+/**
+ * Gulp needs to work in relative paths, NOT absolute like we've configured in
```

Comment:
Grav needs an absolute path for its generated spaces. We're currently using a workaround to achieve what we need for the `twig-namespaces` plugin. See `apps/grav/twig-namespaces.yml` for the paths we need (under `namespaces` key). Eventually the path for this would need to be something like: `user/themes/particle/source/_patterns/01-atoms`

Reply:
Aaaand, solved :) Check out the changes to gulp-twig-namespaces.js and gulpfile.js